src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  gap: 48px 46px;
+  grid-template-columns: repeat(var(--column-count), 1fr);
+  justify-items: center;
+
+  @include screen-xs {
```

Comment:
The mixin `screen-xs` is used here. Ensure that this mixin is defined in your SCSS setup to avoid compilation errors. It should handle media queries for extra small screens.